app/connectors/CustomsDeclareExportsConnector.scala

```diff
@@ -117,25 +117,30 @@ class CustomsDeclareExportsConnector @Inject()(appConfig: AppConfig, httpClient:
 
   def submitDeclaration(id: String)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Submission] =
     httpClient
-      .POSTEmpty[Submission](s"${appConfig.customsDeclareExportsBaseUrl}${appConfig.declarations}/$id/submission")
+      .POSTEmpty[Submission](s"${appConfig.customsDeclareExportsBaseUrl}/submission/$id")
+
+  def fetchSubmissions()(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Seq[Submission]] =
+    httpClient.GET[Seq[Submission]](s"${appConfig.customsDeclareExportsBaseUrl}${appConfig.submissions}").map { response =>
```

Comment:
Why using here `${appConfig.submissions}` and using instead `/submission` at line 120 ? Or viceversa.
I get they are different EPs. Only why the different syntax?

Reply:
The only reason is legacy. Initially, EPs were configurable, because we assumed we may want or need to change them in different environments. I don't think this is the case anymore, but the code stays.
If you think it is a good idea to stop using them via `AppConfig`, then I am happy to refactor this part.

Reply:
What I would suggest is to choose one single approach for all EPs and not a mixed one.

Personally I like to have in `application.conf` `customsDeclareExportsBaseUrl` and the rest of the path directly specified in the connector, because to have it in `application.conf` does not add any value, is less readable and make the impl more complicated. And in any case you have to redeploy if paths need to be modified. I would have paths in `application.conf` only if they were used in multiple places.

But, as long as we follow for all EPs the same approach, I'm fine even to have paths in `application.conf`.